packages/components/style.css

```diff
@@ -130,14 +107,7 @@ html {
 
 /* fix for nextra navbar for HiveNavigation */
 :root {
-  --nextra-navbar-height: 90px;
-}
-
-@media (min-width: 768px) {
-  .nextra-sidebar-container,
-  .nextra-toc > .nextra-scrollbar {
-    top: var(--nextra-navbar-height);
-  }
+  --nextra-navbar-height: 82px;
```

Comment:
What changed by 8px? Some margin or the HiveNavigation height?

Reply:
Height of hive navbar is 82 or 83px with a border 
<img width="1654" alt="image" src="https://github.com/user-attachments/assets/9f9a52ce-9e71-4f44-a77a-fd5eb478a393">